visitor-sdk/_methods.md

```diff
@@ -368,7 +368,7 @@ visitorSDK.sendTicketForm({
 
 ## disconnect
 
-Disconnect Vistior SDK. Visitor won't be tracked, you won't be notified about agent's availability status change. You will be automatically connected again after using sendMessage or setVisitorData methods. 
+Disconnect Visitor SDK. Visitor won't be tracked, you won't be notified about agent's availability status change. You will be automatically connected again after using sendMessage or setVisitorData methods. 
```

Comment:
A visitor won't be tracked, and you won't be notified about the changes of agent's availability status.